core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTableBuilder.java

```diff
@@ -48,23 +48,39 @@ public CarbonTableBuilder tablePath(String tablePath) {
     return this;
   }
 
+
+  public CarbonTableBuilder isUnManagedTable(boolean isUnManagedTable) {
+    Objects.requireNonNull(isUnManagedTable, "UnManaged Table should not be null");
+    this.unManagedTable = isUnManagedTable;
+    return this;
+  }
+
   public CarbonTableBuilder tableSchema(TableSchema tableSchema) {
     Objects.requireNonNull(tableSchema, "tableSchema should not be null");
     this.tableSchema = tableSchema;
     return this;
   }
 
+  public CarbonTableBuilder setUUID(long uuid) {
```

Comment:
UID cannot be in Tablelevel datastructure, as it is unique for one load. Move it to LoadModel

Reply:
Removed